### ISSUE TITLE
[5.6.6] Server crash on startup

### ISSUE THREAD

**Issue by CplPibald** [kreezxil/Lava-Dynamics#22]
lavadynamics-1.12.2-5.6.6
forge-1.12.2-14.23.5.2838

Existing world using dedicated server (openjdk on Debian)
Upgrade Lava Dynamics from 5.5.15 to 5.6.6
Server crashes on startup

```
Description: Exception in server tick loop

java.lang.NoClassDefFoundError: net/minecraft/client/entity/EntityPlayerSP
        at com.eleksploded.lavadynamics.LavaDynamics.playerJoin(LavaDynamics.java:109)
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_376_LavaDynamics_playerJoin_EntityJoinWorldEvent.invoke(.dynamic)
        at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
        at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
        at net.minecraft.world.WorldServer.func_175650_b(WorldServer.java:1065)
```

[Full crash log](https://paste.ubuntu.com/p/3hm6gGT3bz/)

LD_CheckedStorage and LD_VolcanoStorage are both empty files (0 bytes).

**Comment by kreezxil** [comment]
As i understand it from my programmer that when upgrading the mod from
5.5.15 to 5.6.anything you will have to wipe your world because of a change
in how data is stored in regards to the chunks.
Therefore the easiest way to resolve the issue is to reverse the update.
Yet, that is not ideal. The ideal is to wipe your world so that you benefit
from the even less buggier volcanic behavior.

**Comment by eleksploded** [comment]
This is true, but this crash was unrelated. It was fixed in 5.6.7.
